edx_proctoring/templates/proctoring/seq_timed_exam_ready_to_submit.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="sequence timed-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
```

Comment:
I will not make suggestions in this template for this round: not for MVP correct?